content/bn/blog/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: কুবারনেটস ব্লগ
+linkTitle: ব্লগ
+menu:
+  main:
+    title: "ব্লগ"
+    weight: 40
+    post: >
+       <p>কুবারনেটেস এবং সাধারণভাবে কন্টেইনার স্পেসের জন্য সাম্প্রতিক সংবাদ পড়ুন, এবং প্রেস থেকে সরাসরি প্রাপ্ত প্রযুক্তিগত হাউ-টুস পেতে থাকুন।</p>
```

Comment:
কুবারনেটিস এবং সাধারণভাবে কন্টেইনার স্পেসের জন্য সাম্প্রতিক সংবাদ পড়ুন, এবং প্রেস থেকে সরাসরি প্রাপ্ত প্রযুক্তিগত কিভাবে-কী  পেতে থাকুন।

Reply:
fixed it @asem-hamid do check